op-e2e/e2eutils/disputegame/cannon_helper.go

```diff
@@ -48,9 +48,17 @@ func (g *CannonGameHelper) CreateHonestActor(ctx context.Context, rollupCfg *rol
 	}
 }
 
-func createConfigOption(t *testing.T, rollupCfg *rollup.Config, l2Genesis *core.Genesis, gameAddr common.Address, l2Endpoint string) challenger.Option {
+func createConfigOption(
+	t *testing.T,
+	rollupCfg *rollup.Config,
+	l2Genesis *core.Genesis,
+	factoryAddr common.Address,
+	gameAddr common.Address,
+	l2Endpoint string,
+) challenger.Option {
```

Comment:
Is this from a gofmt from a golang lint or did we settle on some config?

Reply:
This method definition got another option which made it even longer and so I manually wrapped it. So following the existing "use common sense" policy.